yellowbrick/cluster/elbow.py

```diff
@@ -99,7 +99,7 @@ def distortion_score(X, labels, metric='euclidean'):
         distances = distances ** 2
 
         # Add the mean square distance to the distortion
```

Comment:
Thank you for making this change - would you mind also updating the comment above to reflect the new sum? I know it's small - but I just want to make sure we're consistent!